lib/preprocessor.js

```diff
@@ -84,10 +84,12 @@ function createCoveragePreprocessor (logger, basePath, reporters = [], coverageR
 
   // if coverage reporter is not used, do not preprocess the files
   if (!reporters.includes('coverage')) {
+    log.debug('coverage not included in reporters %s', reporters)
```

Comment:
Seems like this ought to be `log.info()` right

Reply:
I thought it's same log level as https://github.com/karma-runner/karma-coverage/blob/c69e003f3c393c1146309db4a9693ce0e872c3a1/lib/preprocessor.js#L113. Either level is acceptable to me, though.

Reply:
The difference I see is that echoing the file.originalPath is "Normal operation" so we only emit it when we are debugging. The log line is also per input file.

As I understand it, in this case the issue is essentially an error since the karma-coverage plug was added but disabled by not including a reporter.  And the log line is once per run.  Hence info()

I wonder if this path ought to simply exit the process.

Reply:
karma-coverage plugin is added automatically. I think coverage being excluded from reporter is reasonable if karma is launched in watch mode as shown in karma.conf.js in #395. That karma.conf.js file was generated from yeoman template so I believe the setting has some popularity.